kolibri/utils/cli.py

```diff
@@ -1,11 +1,17 @@
 from __future__ import absolute_import, print_function, unicode_literals
 
+from threading import Thread
```

Comment:
Did you run `pre-commit install` etc? It's not a flake8 thing, it's an isort thing...